.github/workflows/lint.yml

```diff
@@ -21,7 +21,7 @@ jobs:
       matrix:
         # Ignore nightly for now, it fails too often
         #rust_version: ["nightly", "1.60.0", "stable"]
-        rust_version: ["1.60.0", "stable"]
+        rust_version: ["1.63.0", "stable"]
```

Comment:
the prost-build dependency will autogenerate spans annotated with `#[allow(clippy::derive_partial_eq_without_eq)]`

this lint rule doesn't exist in rust versions <1.63.0, so the workaround was to pin prost-build to an older version.

I think just bumping our rust_version we use is a better solution, but let me know if there are objections to this or implication's im missing

Reply:
I opened a PR to bump it to 1.64.0 and all the tests at that time passed. Additionally, the PHP profiler has been building and shipping with Rust 1.64.0 for a while now. I recommend moving to 1.64.0.

Reply:
sounds good, I'll remove the changes here and wait for your PR to merge